### ISSUE TITLE
Unexpected results on custom input scan

### ISSUE THREAD

**Issue by xiyichen** [boqian-li/ETCH#4]
Hello,
Nice work! I was trying out the pretrained all-in-one model on input scans from the THuman 2.0 data.
I ran the following command:
`CUDA_VISIBLE_DEVICES=0 python src/inference_demo.py --scan_path ./gt_mesh_0009.obj --gender male --model_path ./all-in-one.pth`
on this input mesh:
<img width="425" height="641" alt="Image" src="https://github.com/user-attachments/assets/cc28febb-7a7d-4529-a63b-8aa954b92255" />
but the predicted SMPL mesh does not look good:
<img width="493" height="747" alt="Image" src="https://github.com/user-attachments/assets/b6f2dd23-0bc0-40aa-8b69-56843ede7694" />
The final optimization loss for both stages are around 0.12.
I'm wondering if there's something I missed to get expected results. Thank you!

**Comment by boqian-li** [comment]
Hi, thanks for your interest in our work! 
I remember some scans in THUman are not at the right scale. For example, the expected height of the input scan should be approximately 1.6 to 1.8, at least around this range. Could you please provide more information about that?

**Comment by xiyichen** [comment]
Thanks for your quick reply. After centering, it seems the vertices are within [-0.45, 0.45], which is indeed not at right scale:
<img width="581" height="85" alt="Image" src="https://github.com/user-attachments/assets/0b3ea53c-a242-4a7c-8c65-1b3634586338" />

However, when I rescale it to [-0.85, 0.85], so that the height is around 1.7, the result did not improve much:

<img width="352" height="519" alt="Image" src="https://github.com/user-attachments/assets/82f1a4c1-facc-4261-8f22-dc002faa5ebf" />

**Comment by xiyichen** [comment]
Ah, I found that the orientation of my mesh was also not matching the training samples. After rotating the mesh and scaling it properly, I'm getting much better results now. Thank you!

<img width="231" height="443" alt="Image" src="https://github.com/user-attachments/assets/a6ee4c61-a50f-4f7b-994c-a40ff2e09dc3" />

**Comment by boqian-li** [comment]
Happy to hear that :)

**Comment by xiyichen** [comment]
@boqian-li I have another question. How did you select the markers and obtain their indices from the SMPL vertices? I'd like to finetune the model on datasets like THuman which only provide SMPL-X fittings, so I'm wondering how I could similarly obtain the marker indices for SMPL-X.

**Comment by boqian-li** [comment]
Hi, we determine the marker indices following the [moshpp](https://github.com/nghorbani/moshpp) and [soma](https://github.com/nghorbani/soma). You can use this file [markers86_smplx.json](https://github.com/user-attachments/files/21824339/markers86_smplx.json) for smpl-x.

**Comment by xiyichen** [comment]
Thank you!
